data/user_data_base.ps2

```diff
@@ -199,6 +199,12 @@ function Get-MyName {
     return $nameTag.Value
 }
 
+function Get-AppNames {
+    $instanceId = (New-Object System.Net.WebClient).DownloadString("http://169.254.169.254/latest/meta-data/instance-id")
+    $appNameTag =  Get-EC2Tag | ` Where-Object {$_.ResourceId -eq $instanceId -and $_.Key -eq 'AppNames'}
```

Comment:
Does the ` belong there?  I guess you probably took out the line break and it doesn't actually matter, but thought I'd mention it.